README.md

```diff
@@ -28,7 +28,8 @@ UPSTREAM_CHE_PATH=/path/to/upstream/che
 mvn -DwithoutKeycloak                          `# disable Keycloak support` \
     -Pfast                                     `# skip tests and other verifications` \
     -PmultiThread                              `# enable maven multi threading` \
-    -DlocalCheRepository=${UPSTREAM_CHE_PATH}  `# get already built upstream Che` \
+    -DlocalCheRepository=${UPSTREAM_CHE_PATH}  `# get already built upstream Che local folder` \ 
```

Comment:
I don't see why the `-P '!checkout-base-che'` is necessary here, since when using a local upstream Che repo (with `-DlocalCheRepository`) the upstream Che will never be checked-out from Git (to `target/export`) and rebuilt.
These 2 options are related to 2 different use-cases that are mutually exclusive.